lib/naughty/layout/box.lua

```diff
@@ -378,18 +379,20 @@ local function new(args)
     end
 
     --TODO remove
-    local function hide()
-        local n = ret._private.notification[1]
+    local function hide(reason)
```

Comment:
`reason` seems to be unused

Reply:
@actionless yes good spot, should have been passed to `destroy`. Updated.